### ISSUE TITLE
when cohort, concept set is opened and then closed, ATLAS resets the last page that was active

### ISSUE THREAD

**Issue by gklebanov** [OHDSI/Atlas#760]
When paging through concepts, cohorts or other design artifacts and concept set or cohort is open, the page that was current is lost and ATLAS goes back to page 1. Instead, the current page should be kept

**Comment by pbr6cornell** [comment]
I agree this is only an 'enhancement', not 'bug', but I think it definitely falls squarely in the 'performance' territory of ATLAS 2.8, because it costs a lot of time to page through a list repeatedly to get back to where you were, and not having to do that would save a good bit of user time.

**Comment by anthonysena** [comment]
I'm moving this item to 3.0 - the changes required for the underlying data table component are too complex and introduce other problems that we can better sort out with a new table component.
